src/Amazon.Extensions.CognitoAuthentication/CognitoUser.cs

```diff
@@ -359,6 +359,7 @@ public virtual async Task SetUserSettingsAsync(IDictionary<string, string> userS
         /// <param name="limit">Maxmimum number of devices to be returned in this call</param>
         /// <param name="paginationToken">Token to continue earlier search</param>
         /// <returns>Returns a list of CognitoDevices associated with this user</returns>
+        [Obsolete("This method is depreceated since it doesn't return paginationToken. Use ListDevicesWithPaginationTokenAsync instead to access underlying ListDevicesResponse.")]

```

Comment:
typo `depreceated` -> `deprecated`

My suggest for this is
```
This method is deprecated since it only lists the first page of devices. The method ListDevicesWithPaginationTokenAsync should be used instead which allows listing additional pages of devices.
```

Reply:
Fixed in latest commit.